ale/base/type_sensor.py

```diff
@@ -162,7 +162,7 @@ def ephemeris_stop_time(self):
         : double
           Center ephemeris time for an image
         """
-        return self.ephemeris_start_time + (self.interframe_delay) * (self.num_frames - 1) + self.exposure_duration
+        return self.ephemeris_start_time + (self.interframe_delay * self.num_frames)
```

Comment:
Same here

Reply:
Yes, the math here was weird relative to ISIS